docs/integration_resources/software/resolve.md

```diff
@@ -1,5 +1,75 @@
-# Resolve
+# Davinci Resolve
 
-Shift integration in Resolve is on the works! 
+## Running Shift in Davinci Resolve
+
+Shift can be integrated in Davinci Resolve and used to automate editorial process using all the catalog provided with Shift, but specially the Shift_Resolve catalog.
+
+<figure>
+      <img src="images/resolve_shift_ui.png" alt="Resolve Shift UI">
+      <figcaption><b>Figure 1</b>: Shift UI in Davinci Resolve.</figcaption>
+</figure>
+
+To run Shift inside Davinci Resolve it is required to set up the configuration needed by Blackmagic to be able to execute Python code inside Resolve. 
+
+Blackmagic usually provides a `Readme.txt` file where these requirements are detailed for the version in use in the installation folder `C:\ProgramData\Blackmagic Design\DaVinci Resolve\Support\Developer\Scripting\Readme.txt`
+
+Apart from the official requirements, the `PYTHONHOME` environment is required to be set. It must point to the Python interpreter to be used in Resolve.
+
+To use the Python interpreter provided with Shift, set up the variable in this way:
+
+**PYTHONHOME** : <path_to_your_shift_folder>/shift/thirdparty/python
+
+It's required to add to the PATH the directories of Shift too. This paths can be added in the environment, before open Resolve, or in the python file where Shift will be executed.
+
+```
+PATH: <path_to_the_shift_installation_folder>
+PATH: <path_to_the_shift_installation_folder>/shift/thirdparty/python/Lib/site-packages
+
+```
+
+## Shift Menu
+
+The Resolve Script menus for Python files are limited to add entries to the section of `Workspace > Scripts` and splitted by workspace layouts inside.

```

Comment:
This one should probably be reviewed by @JorgeHernandezibanezinbibo 

```suggestion
The Davinci Resolve Script menus can add entries to the section of `Workspace > Scripts`.

```

Reply:
@albertochimentiinbibo what's the problem with the original? I'm not sure if I agree with the suggestion.
```suggestion
The Davinci Resolve Script menus for Python files are limited to the section of `Workspace > Scripts` and splitted by workspace layouts inside (1 folder per layout).

```

The workspace layout part is importand because depending in what folder do you add Shift, Shift will appear in a different Resolve layout. The menus are not the same in the Edit layout and the cut or FX layout for example. Make sense?

Reply:
I think that the sentence is not clear enough to understand what you refer to.
Do you mean that Python files that define a menu should be added in the `Workspace > Scripts` section?
And that this section is divided by folders per layout?
I think the confusing word here is "limited". What do you mean with limited?

Reply:
I mean that the only way to add a menu is to add a Python file to make it apper like an entry in `Workspace > Scripts` there are not other way to create a python menu in Davinci Resolve. And inside that, is divided by folders per layout, yes.

Reply:
Ok, what do you think about this?
```suggestion
Custom sub-menus in DaVinci Resolve are defined by Python files that are added as entries to the `Workspace -> Scripts` menu section. This section is divided into folders, with each one corresponding to a workspace layout.
```

Reply:
For me it´s ok. @albertochimentiinbibo ?

Reply:
yes please commt it